src/services/CurrencyService.js

```diff
@@ -8,7 +9,7 @@ export const getCurrencyValues = () => {
   ])
   .then(tables =>  {
     return {
-      rates: [...tables[0][0].rates, ...tables[1][0].rates],
+      rates: [...tables[0][0].rates, ...tables[1][0].rates, polishCurrency],
```

Comment:
and maybe we can sort it here. When the `rates` value is set. Because even if we use this service in any other place, it should be sorted.

Reply:
You are right. I change it.